tcl/auth.tcl

```diff
@@ -76,7 +76,7 @@ proc qc::auth_session { session_id } {
     #| Try to authenticate user based on the session_id given
     #| Return the user_id if successful
     #| On failure throw AUTH error
-    if { [session_exists $session_id] } {
+    if { [session_valid $session_id] } {
         return [session_user_id $session_id]
```

Comment:
@bvw Is this what we want this will enforce a default age limit of 12 hours and idle_timeout of 1 hour for all projects, even TLC ?